packages/lit-node-client-nodejs/src/lib/helpers/validate-bls-session-sig.ts

```diff
@@ -0,0 +1,28 @@
+import { AuthSig } from '@lit-protocol/types';
+import { uint8arrayToString } from '@lit-protocol/uint8arrays';
+import { ethers } from 'ethers';
+
+const LIT_SESSION_SIGNED_MESSAGE_PREFIX = 'lit_session:';
+
+export const blsSessionSigVerify = (
```

Comment:
```suggestion
/**
 * Verifies a BLS session signature.
 *
 * @param {Function} verifier - A wasm function that takes a public key, message, and signature to verify. NOTE: `public_key` is snake cased because it's a wasm parameter
 * @param {string} networkPubKey - The public key of the network.
 * @param {AuthSig} authSig
 * @typedef {Object} AuthSig
 * @property {string} sig - The signature in string format.
 * @property {string} signedMessage - The message that was signed.
 */
export const blsSessionSigVerify = (
```